packages/3dom/src/context-host.ts

```diff
@@ -0,0 +1,108 @@
+import {ThreeDOMCapability} from './api.js';
+import {ThreeDOMExecutionContext} from './context.js';
+import {ThreeDOMMessageType} from './protocol.js';
+
+// This fake worker class is just used to capture the transfer of the message
+// port.
+class FakeWorker extends EventTarget implements Worker {
+  onerror: EventListener|null = null;
+  onmessage: EventListener|null = null;
+  onmessageerror: EventListener|null = null;
+  port2: MessagePort|null = null;
+  hostExecutionContext: HostThreeDOMExecutionContext;
+
+  constructor(hostExecutionContext: HostThreeDOMExecutionContext) {
+    super();
+    this.hostExecutionContext = hostExecutionContext;
+  }
+
+  postMessage(message: any, transfer: Array<Transferable>): void;
```

Comment:
what type is message? it seems like it is just an Object? Linter is complaining we're supposed to avoid any in general...